### DISCUSSION TITLE
Error in Pimcore Academy courses

### DISCUSSION THREAD

**Discussion by irojasfs** [pimcore/pimcore#10360] [General]
In the last few days, I have noticed that in some courses of the academy, when trying to watch a video, the section keeps loading since the course page is loaded and it is impossible to watch the video. I have tried different browsers and I have the same problem.

Does anyone have any idea how to solve this problem or is it a site problem?

Here are some screenshots of the issue

![imagen](https://user-images.githubusercontent.com/87441114/134071157-e643c2c8-dea4-4cc5-8c2f-740ab9f9585f.png)

![imagen](https://user-images.githubusercontent.com/87441114/134070624-80f59521-4080-40ee-ae35-5376635e8c1d.png)

![imagen](https://user-images.githubusercontent.com/87441114/134071084-564e9e7a-65c0-43a7-86a0-1de7fe170dd8.png)

**Comment by fashxp** [comment]
Hi, 
is it still the case with some videos? Last week we had some technical problems, but they should be fixed now. 
Please share the courses, if there are still problems... Thank you...

BR
Christian

**Reply by irojasfs** [reply]
Hi
the problem keeps happening, some courses where I have seen the problem are:

Course 'Pimcore PIM'
Course 'Pimcore DAM '
Course 'Pimcore DXP & CMS'.
Course 'Pimcore Enterprise Subscription'

in the 'Pimcore Consultant' Certificate section and in this section it is more frequent to see this problem
but I have seen that other courses and certificate section sometimes randomly also have the same problem.

**Reply by fashxp** [reply]
Hmm, just clicked through all of them, were fine for me. Could you retry please (eventually clear browser cache)...

BR
Christian

**Reply by NiklasBr** [reply]
@fashxp the academy site has been slow the past week or so, sometimes loading a page has taken a full minute.

**Reply by fashxp** [reply]
@NiklasBr can you name some page types which are slow? anyhow reproducible, or just 'random'? 
on tuesday we made some improvements, maybe they solved it already?

**Reply by NiklasBr** [reply]
It is much faster today for my part. It was most pages, especially those with video files that were slow.

**Reply by skynetindia** [reply]
Hey,

Does this problem get resolved because I m facing the same (too much loading while watching the Academy videos)

FYI I've been seeing "Why Pimcore" course but after certain time I cannot see anything as it show loading only

**Reply by fashxp** [reply]
hmm, works fine for me now. 
can you try an incoginto-tab to see if it has something to do with your session?

**Reply by skynetindia** [reply]
Just FYI I have check multiple devices, multiple internet connections and as per your comment check the same on incognito mode 

But after 1:20min that it is showing loading only 

I have stayed there for more than 1hour but still, it is been showing the loading only

**Reply by fashxp** [reply]
when showing the why pimcore video?

**Reply by fashxp** [reply]
any errors in the browser console?

**Reply by skynetindia** [reply]
I have tried to check multiple videos, why pimcore(Pimcore overview, pimcore case studies), getting started course(System Requirements, Installing pimcore)
But after showing for couple of minutes it shows loading only

**Reply by skynetindia** [reply]
> any errors in the browser console?

No there is no error in the console
